tests/foreman/cli/test_katello_agent.py

```diff
@@ -16,99 +16,89 @@
 
 :Upstream: No
 """
-import time
-
 import pytest
-from broker import VMBroker
 
-from robottelo.api.utils import wait_for_errata_applicability_task
-from robottelo.cli.activationkey import ActivationKey
-from robottelo.cli.factory import make_activation_key
-from robottelo.cli.factory import make_host_collection
+from robottelo import constants
 from robottelo.cli.factory import setup_org_for_a_custom_repo
 from robottelo.cli.factory import setup_org_for_a_rh_repo
-from robottelo.cli.host import Host
-from robottelo.cli.hostcollection import HostCollection
 from robottelo.config import settings
-from robottelo.constants import FAKE_0_CUSTOM_PACKAGE_GROUP
-from robottelo.constants import FAKE_0_CUSTOM_PACKAGE_GROUP_NAME
-from robottelo.constants import FAKE_0_CUSTOM_PACKAGE_NAME
-from robottelo.constants import FAKE_1_CUSTOM_PACKAGE
-from robottelo.constants import FAKE_1_CUSTOM_PACKAGE_NAME
-from robottelo.constants import FAKE_2_CUSTOM_PACKAGE
-from robottelo.constants import FAKE_2_CUSTOM_PACKAGE_NAME
-from robottelo.constants import PRDS
-from robottelo.constants import REPOS
-from robottelo.constants import REPOSET
-from robottelo.hosts import ContentHost
+from robottelo.helpers import InstallerCommand
+
+
+pytestmark = [
+    pytest.mark.run_in_one_thread,
+    pytest.mark.skip_if_not_set('clients', 'fake_manifest'),
+]
+
 
+@pytest.fixture(scope='module')
+def sat_with_katello_agent(module_destructive_sat):
```

Comment:
Rename this to `destructive_sat_with_katello_agent` and move this to global fixtures alongside `module_distructive_sat` so that others would like to use it :)

Reply:
As katello-agent is deprecated after 6.10, so not sure if anyone would like to use this fixture other than these tests, so it makes no sense to move to global fixtures, and now I think we should encourage using rex here instead of katello-agent for such operations, still, if we get any explicit requirement to use this, so we can surely move this fixture :)

Reply:
@Gauravtalreja1 Agree !